.github/workflows/ci.yml

```
@@ -27,9 +27,9 @@ jobs:
           extensions: mysql
       - name: Get composer cache directory
         id: composer-cache
-        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+        run: echo "dir=$(composer config cache-files-dir | tr -d '\n' | tr -d '\r')" >> $GITHUB_OUTPUT
       - name: Cache composer dependencies
-        uses: actions/cache@v2
+        uses: actions/cache@v3
```

```suggestion
        uses: actions/cache@v4
```

FYI the v4 is available too, hopefully it doesn't break anything

---

ok, now let it stay as it is